packages/core/parcel-bundler/src/assets/ElmAsset.js

```diff
@@ -129,6 +131,19 @@ class ElmAsset extends Asset {
       return result.code;
     }
   }
+
+  generateErrorMessage(err) {
+    // For some reason, if not converted to a plain object,
+    // the error message is lost somewhere between Pipeline.js
+    // and Bundler.js.
```

Comment:
This fixes an issue for Elm, but I probably also need to do something like this in `Pipeline.js` for other asset types that throw `Error`s.  I'm not certain why this is necessary, but I suspect that there's an object copy somewhere in `WorkerFarm` that causes data to be lost if `Pipeline.process` returns an obejct containing an `Error`

Reply:
This is fixed.